docs/content/2.frontier/5.libraries/2.standard-library.md

```diff
@@ -1,2 +1,57 @@
-# Standard Library
+# Frontier Standard Library
 
+## @rdfrontier/stdlib
+
+A collection of functions/logic that are written in pure TypeScript with limited dependencies. These functions expose generic concepts available for use throughout different projects.
+
+     
+
+## Table of Contents
+
+- Installation
+- Usage
+- Options
+       
+
+## Installation
+
+```
+$ yarn add --save--dev @rdfrontier/stdlib
+```
+
+or
+
+```
+$ npm install --save--dev @rdfrontier/stdlib
+```   
+
+## Usage
+
+```
+Eg: import { log } from '@rdfrontier/stdlib';
```

Comment:
Maybe use a "$" instead of "Eg:" to keep it consistent with the other code snippets.